test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Linux/AcceptanceTest.cs

```diff
@@ -19,15 +19,15 @@
 using Microsoft.Extensions.Options;
 using Microsoft.Extensions.Time.Testing;
 using Microsoft.Shared.Instruments;
-using Microsoft.TestUtilities;
 using Xunit;
 
 namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring.Linux.Test;
 
+[PlatformSpecific(TestPlatforms.Linux)]
 public sealed class AcceptanceTest
 {
     [ConditionalFact]
-    [OSSkipCondition(OperatingSystems.Windows | OperatingSystems.MacOSX, SkipReason = "Linux specific tests")]
+    [PlatformSpecific(TestPlatforms.Linux)]
```

Comment:
When `[PlatformSpecific(TestPlatforms.Linux)]` is applied to the whole class there is no need to repeat it for every test. Moreover, such tests don't need `[ConditionalFact]`, `[Fact]` is just enough. Please fix it for all the occurrences of the same problem.

```suggestion
```